src/main/java/com/github/vkorobkov/jfixtures/loader/FixtureValue.java

```diff
@@ -24,6 +26,11 @@ public FixtureValue(Object value) {
         }
     }
 
+    @XmlValue
+    public String getStringValue() {
```

Comment:
Can it serialize if the method is defined like `Object getValue()`?

Reply:
`fixturesResult.asString() == EXPECTED_XML
|              |
|              java.lang.NullPointerException
com.github.vkorobkov.jfixtures.fluent.impl.XmlJFixturesResultImpl@548a24a


	at com.github.vkorobkov.jfixtures.fluent.impl.XmlJFixturesResultImplTest.processes fixtures to string(XmlJFixturesResultImplTest.groovy:57)`